examples/chemicalEnvironments/README.md

```diff
@@ -0,0 +1,177 @@
+# Alpha Atom SMIRKS: [#8X2H1;R0]
+
+beta_ORtypes = [('#1', [])]
+beta = angle.addAtom(alpha, newORtypes = beta_ORtypes)
+print("Beta Atom SMIRKS: %s" % beta.asSMIRKS())
+# Beta Atom SMIRKS: [#1]
+
+print("New overall SMIRKS: %s" % angle.asSMIRKS())
+# New overall SMIRKS: [#6X3,#7X2:1]~;@[#8,#7+0X3;R1:2]~;@[#6X3,#7:3]~;!@[#8X2H1;R0]~[#1]
+
+removed = angle.removeAtom(beta)
+print("The hydrogen beta to atom3 was remove: ", removed)
+# The hydrogen beta to atom3 was remove:  True
+print("Updated SMIRKS string: %s" % angle.asSMIRKS())
+# print("Updated SMIRKS string: %s" % angle.asSMIRKS())
+```
+
+### Other ChemicalEnvironment Methods
+
+0. Selecting atoms and bonds
+```python
+# when given an integer selectAtom or selectBond returns that atom or bond with that index 
+# otherwise it returns a random atom or bond that meets the specified requirement 
+# If no atom or bond with that type is found then None is returned
+alpha = angle.selectAtom('alpha')
+atom4 = angle.selectAtom(4) # None, angle has 3 indexed atoms
+beta_bond = angle.selectBond('beta') # bond between an alpha and beta atom
+```
+
+1. Getting information about an atom or bond in an environment (i.e. isAlpha returns a boolean)
+```python
+# Check if the alpha atom above is any of the following
+angle.isIndexed(alpha) # False
+angle.isUnindexed(alpha) # True 
+angle.isAlpha(alpha) # True
+angle.isBeta(alpha) # False
+# NOTE - These methods can take an atom or a bond as an argument
+```
+2. Get atoms or bonds in each type of position, for example getIndexedAtoms or getAlphaBonds
+```python
+# We will print the SMIRKS for each indexed atom:
+all_atoms = angle.getAtoms()
+indexed_list = angle.getIndexedAtoms() # list of indexed atoms
+unindexed_list = angle.getUnindexedAtoms() # list of atoms with no index
+alpha_list = angle.getAlphaAtoms()
+beta_list = angle.getBetaAtoms
+# Note - Atoms can be replaced with Bonds in each of the above examples
+```
+3. Report the minimum order of a bond with Bond.getOrder
+```python
+bond1 = angle.selectBond(1)
+bond1.getOrder() # returns 1
+```
+4. Report the valence and bond order around an atom can be reported with getValence and getBondORder
+```python
+atom3 = angle.selectAtom(3)
+angle.getValence(atom3) # number of neighbors atom 3 has
+angle.getBondOrder(atom3) # total bond order around atom 3
+```
+5. Get a bond between two atoms (or determine if the atoms are bonded) with getBond(atom1, atom2)
+```python
+# Check for bonds between each pair of indexed atoms
+A = 1
+B = 2
+atomA = angle.selectAtom(A)
+atomB = angle.selectAtom(B)
+bondAB = angle.getBond(atomA, atomB) # returns None if no bond found
+```
+6. Get atoms bound to a specified atom with getNieghbors
+```python
+# get the neighbors for each indexed atom
+atomA = angle.selectAtom(A)
+neighbor_list = angle.getNeighbors(atomA) 
+```
+
+### `using_environments.ipynb`
+
+This notebook should be your first stop for interactively understanding and using ChemicalEnvironemnts. 
```

Comment:
(Note that some of these typos -- though not this one -- also show up in your `using_environments.ipynb` file and should be corrected there too.)